src/data/real/nnreal.lean

```diff
@@ -49,6 +50,39 @@ This file defines `ℝ≥0` as a localized notation for `nnreal`.
 
 open_locale classical big_operators
 
+namespace real
+variables {ι : Sort*} {f : ι → ℝ}
+
+open set
+
+lemma supr_of_not_bdd_above (hf : ¬ bdd_above (range f)) : (⨆ i, f i) = 0 := Sup_of_not_bdd_above hf
+lemma infi_of_not_bdd_below (hf : ¬ bdd_below (range f)) : (⨅ i, f i) = 0 := Inf_of_not_bdd_below hf
```

Comment:
Please move these lemmas to appropriate files. These two should be right below `Sup_of_not_bdd_above`/`Inf_of_not_bdd_below` and the next two should be in `data.set.pointwise`.